### PR TITLE
Update depenency to avoid CVE-2022-4904

### PR DESCRIPTION
**Pull request by kse3hi**
# Description

Update packages:
* devenv-github-workflows@v2.0.9
* devenv-devcontainer-setup@v1.1.9
* devcontainer-base-image@v1.0.3
* cpp-sdk@v0.3.2

Fix some security issues:
* CVE-2022-4904
* CVE-2023-27536
* CVE-2023-27535
* CVE-2023-27538
* CVE-2023-0466
* CVE-2023-0464
* CVE-2023-0465

## Issue ticket number and link

ADO#15373

## Checklist

* [x] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [x] Vehicle App can process MQTT messages and call the seat service
* [x] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [x] Devcontainer can be opened successfully
* [x] Devcontainer can be opened successfully behind a corporate proxy
* [x] Devcontainer can be re-built successfully

* [x] Release workflow is passing

### CODE REVIEW

File: sdk
Comment (BjoernAtBosch) [inline sdk:1]: Please update to moved label ...

**Review by BjoernAtBosch — CHANGES_REQUESTED** [review]

File: sdk
Comment (kse3hi) [reply sdk:1]: Sorry, forgot to push the latest changes :)

Comment (BjoernAtBosch) [reply sdk:1]: No problem

**Review by BjoernAtBosch — APPROVED** [review]
LGTM

**Pull request merged**